data/gui/window/editor_choose_addon.cfg

```diff
@@ -0,0 +1,233 @@
+#textdomain wesnoth-lib
```

Comment:
Should we have a new rule, "put all new data/gui/... files into `wesnoth` instead of `wesnoth-lib`"? Is that going to be a step in an intended direction, or just adding to the confusion caused by the split?

Reply:
I feel like it doesn't really help much one way or the other. If the split is intended to be resolved, then only making that change to new files won't do anything to resolve it.